web/client/utils/__tests__/GeoStoryUtils-test.js

```diff
@@ -469,4 +471,44 @@ describe("GeoStory Utils", () => {
             expect(parseHashUrlScrollUpdate(url, hash, storyId, null, null)).toBe(url);
         });
     });
+    describe('createWebFontLoaderConfig', () => {
```

Comment:
i would avoided these describe since you are including only one test inside, and an it is sufficient

please remove the describe line from these tests and add the function name inside the it( desctription